### DISCUSSION TITLE
Vercel & GitHub

### DISCUSSION THREAD

**Discussion by Ihbrazil** [vercel/community#4540] [Help] [answered]
### Summary

Olá! Preciso de ajuda pra entrar no Vercel! Não consigo entrar no Vercel... Criação de conta imcompleta... Conta bloqueada... Mas, nem cheguei a entrar no Vercel?

### Steps to Reproduce

Olá! Preciso de ajuda pra entrar no Vercel! Não consigo entrar no Vercel... Criação de conta imcompleta... Conta bloqueada... Mas, nem cheguei a entrar no Vercel?

**Comment by amyegan** [comment, marked as answer]
Sorry you ran into problems with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by Ihbrazil** [comment]
Muitíssimo obrigado!
________________________________
De: Amy ***@***.***>
Enviado: quinta-feira, 19 de outubro de 2023 14:53
Para: vercel/community ***@***.***>
Cc: ihbrazil ***@***.***>; Author ***@***.***>
Assunto: Re: [vercel/community] Vercel & GitHub (Discussion #4540)


Sorry you ran into problems with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

To open a ticket, please use the appropriate direct channel<https://vercel.com/guides/how-to-get-vercel-support#direct-channels> found in this guide: What is the best way to get support from Vercel?<https://vercel.com/guides/how-to-get-vercel-support#direct-channels>
